index.bs

```diff
@@ -311,6 +311,11 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
     that this does not constitute [=user verification=] because [=TUP=], by definition, is not capable of [=biometric
     recognition=], nor does it involve the presentation of a shared secret such as a password or PIN.
 
+: <dfn>Test of User Identity</dfn> 
+: <dfn>TUI</dfn> 
+:: A [=test of user identity=] is a technical process where a user's identity is verified by the authenticator through 
+    performing the <a>user verification</a> process. 
```

Comment:
We should use consistent markup - i.e. always use the [=term=] form instead of `<a>term</a>` unless there is a good reason.